internal/controller/controllers/controller_event_wrapper.go

```diff
@@ -36,10 +36,12 @@ func (c *controllerEventsWrapper) AddEvent(ctx context.Context, clusterID strfmt
 	c.log.Debugf("Pushing cluster event %s %s", cluster.KubeKeyName, cluster.KubeKeyNamespace)
 	c.crdEventsHandler.NotifyClusterDeploymentUpdates(cluster.KubeKeyName, cluster.KubeKeyNamespace)
 	if hostID != nil {
-		// TODO once host will have infraEnv params we need to use common.GetHostFromDB()
-		// till then we will use same namespace as cluster deployment
-		c.log.Debugf("Pushing event for host %q %s %s", hostID, cluster.KubeKeyName, cluster.KubeKeyNamespace)
-		c.crdEventsHandler.NotifyAgentUpdates(hostID.String(), cluster.KubeKeyNamespace)
+		host, err := common.GetHostFromDB(c.db, clusterID.String(), hostID.String())
```

Comment:
We should make only one call to DB, why we can't bring hosts with cluster call?

Reply:
yeah we can, but then we should move KubeKeyNamespace to model.Host instead of common.Host (i.e. it's similar to common.Cluster now). so it would be inconsistent, but maybe indeed worth it to reduce calls.

Reply:
less calls better performance.
@filanov what do you think?

Reply:
Probably better to reduce then - moved KubeKeyNamespace to models.Host and used EagerLoading for the cluster call.

Reply:
I don't think that scale is a problem in this case, we don't have a lot of events for each cluster, i don't think that exposing kube-api in our rest-api model is a good flow. 
Another call for DB to find specific host will probably not become a bottleneck